src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
@MikeGitb thanks for your great review. I've added all of you suggestions but switching to `ScopedConnection` for `ConnectionList`'s container type, as `noexcept` isn't available to us until we switch to Visual Studio 2015 (opened [this issue](https://github.com/cinder/Cinder/issues/1681)). It sounds like using `vector<ScopedConnection>` won't be safe until then.

Everything look good to you?